app/src/main/javascript/bundles/experiment-page/index.js

```diff
@@ -1,20 +1,21 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 
-import ExperimentPageRouter from './src/ExperimentPageRouter'
+import ExperimentPageRouter from '@ebi-gene-expression-group/scxa-experiment-page'
 
 const render = (options, mountNodeId) => {
   ReactDOM.render(
     <ExperimentPageRouter
-      atlasUrl={options.atlasUrl}
-      resourcesUrl={options.resourcesUrl}
-      experimentAccession={options.content.experimentAccession}
-      accessKey={options.content.accessKey}
-      species={options.content.species}
-      tabs={options.content.tabs}
-      anatomogram={options.content.anatomogram} />,
+        atlasUrl={options.atlasUrl}
+        resourcesUrl={options.resourcesUrl}
+        experimentAccession={options.content.experimentAccession}
+        accessKey={options.content.accessKey}
+        species={options.content.species}
+        tabs={options.content.tabs}
+        anatomogram={options.content.anatomogram}
```

Comment:
I can not see `anatomogram` property in the `ExperimentPageRouter` component in the `scxa-experiment-page` package.

Reply:
I don't understand your questions. What do you mean by `property`? Anatomogram is one of the tabs and if you run the webpack server in the `atlas-component`, you may see the content of anatomogram.